Podcasts hosted by Buzzsprout can't be downloaded - "Failed to reach [URL].mp3, status is 403"

**Describe the bug**
Podcasts hosted by Buzzsprout.com will not download, even after repeated tries with different podcasts.

**To Reproduce**
Try this podcast for example: https://feeds.buzzsprout.com/1534219.rss

**Expected behavior**
Episodes to be downloaded normally, as with other podcast hosting sites.

**Example log**
```
[2024-07-07 00:33:14][INFO] Start download
[2024-07-07 00:33:14][INFO] Processing RSS document
[2024-07-07 00:33:15][INFO] 135 files will be downloaded
[2024-07-07 00:33:15][INFO] Saving 2024-07-01 Getting to Know Trans Voices Scott Niswander.mp3 to D:\Podcasts\2024-07-01 Getting to Know Trans Voices Scott Niswander.mp3 from https://www.buzzsprout.com/1534219/15339040-getting-to-know-trans-voices-scott-niswander.mp3
[2024-07-07 00:33:15][ERROR] Failed to reach https://www.buzzsprout.com/1534219/15339040-getting-to-know-trans-voices-scott-niswander.mp3, status is 403
[2024-07-07 00:33:20][ERROR] Failed to reach https://www.buzzsprout.com/1534219/15339040-getting-to-know-trans-voices-scott-niswander.mp3, status is 403
[2024-07-07 00:33:25][ERROR] Failed to reach https://www.buzzsprout.com/1534219/15339040-getting-to-know-trans-voices-scott-niswander.mp3, status is 403
[2024-07-07 00:33:30][ERROR] Download of https://www.buzzsprout.com/1534219/15339040-getting-to-know-trans-voices-scott-niswander.mp3 failed after 3 tries
```

**System Information:**
Windows 11.

--- a/src/__init__.py
+++ b/src/__init__.py
@@ -1,1 +1,1 @@
-pbd_version = '0.8'
+pbd_version = '0.10.0'

--- a/src/bulk_downloader.py
+++ b/src/bulk_downloader.py
@@ -262,7 +262,8 @@ class BulkDownloader:
         if self._is_url():
             try:
                 headers = {'Accept': '*/*',
-                           'User-Agent': f'PodcastBulkDownloader/{pbd_version}' }
+                           'User-Agent':
+                               f'PodcastBulkDownloader/{pbd_version}'}
                 r = requests.get(self._url, headers=headers)
             except requests.RequestException as exc:
                 err_str = 'Failed to connect to URL ({})'.format(exc)

--- a/src/bulk_downloader.py
+++ b/src/bulk_downloader.py
@@ -43,9 +43,10 @@ def download_with_resume(url: str, path: str, cb: Callback = None) -> bool:
 
     if cb and cb.is_cancelled():
         return False
-
+    s = requests.Session()
+    s.headers["User-Agent"] = f"PodcastBulkDownloader/{pbd_version}"
     try:
-        r = requests.head(url, allow_redirects=True)
+        r = s.head(url, allow_redirects=True)
     except requests.exceptions as e:
         logging.error(e)
         return False
@@ -70,8 +71,8 @@ def download_with_resume(url: str, path: str, cb: Callback = None) -> bool:
             if cb and cb.is_cancelled():
                 return False
             resume_header = {'Range': 'bytes=%d-' % last_byte}
-            resume_request = requests.get(url, headers=resume_header, stream=True,
-                                          verify=True, allow_redirects=True)
+            resume_request = s.get(url, headers=resume_header, stream=True,
+                                   verify=True, allow_redirects=True)
             for data in resume_request.iter_content(chunk_size):
                 last_byte += len(data)
                 if cb and cb.is_cancelled():
@@ -261,7 +262,7 @@ class BulkDownloader:
         if self._is_url():
             try:
                 headers = {'Accept': '*/*',
-                           'User-Agent': 'PodcastBulkDownloader v' + pbd_version}
+                           'User-Agent': f'PodcastBulkDownloader/{pbd_version}' }
                 r = requests.get(self._url, headers=headers)
             except requests.RequestException as exc:
                 err_str = 'Failed to connect to URL ({})'.format(exc)

